common/src/warp_runner/manager/commands/raygun_commands.rs

```diff
@@ -297,6 +305,59 @@ pub async fn handle_raygun_cmd(
 
             let _ = rsp.send(r);
         }
+        RayGunCmd::SendMessageForSeveralChats {
+            convs_id,
+            msg,
+            attachments,
+            ui_msg_id: ui_id,
+            rsp,
+        } => {
+            for chat_id in convs_id {
```

Comment:
It might be better to push the logic into a `FutureUnordered` and poll that to completion

Reply:
Looking over the logic of the commands in this module, some changes may need to be done but is outside of the scope of this PR so for now this suggestion can be ignored.